src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,22 @@
+import racingcar.domain.Car
+import racingcar.domain.Cars
+import racingcar.domain.RandomMovingStrategy
+import racingcar.view.InputView
+import racingcar.view.ResultView
+
+fun main() {
+    // Input
+    val quantity = InputView.inputCarQuantity()
+    val tryCount = InputView.inputTryCount()
+
+    // Create cars
+    val strategy = RandomMovingStrategy()
+    val carList = mutableListOf<Car>()
+    for (i in 1..quantity) {
+        carList.add(Car(strategy))
+    }
```

Comment:
코틀린에서는 사용자 편의를 위해 다양한 [확장 함수](https://kotlinlang.org/docs/extensions.html), [편의 메서드](https://kotlinlang.org/api/latest/jvm/stdlib/)들을 제공해주고 있어요.

덕분에 위 경우는 다음과 같이 작성해볼 수도 있는데요!

```suggestion
   val carList = List(quantity) { Car(strategy) }
```

<br/>

"코틀린에서 이미 다양한 편의 메서드들을 제공해주고 있기에 조금 찾아보면 간단하게 작성할 수 있다." 정도를 말씀드리고 싶어서 코멘트 남겨보아요! 💭